src/components/SelectComponents/SelectBase/SelectBase.scss

```diff
@@ -34,6 +34,10 @@
         margin-top: -1px;
     }
 
+    &__list--focused:empty {
+        border-top: none;
```

Comment:
I think this rule should be applied to `SelectBase__list:empty` not `SelectBase__list--focused:empty`.

Because now, when we focus on an input, its height shrinks by 1px.

https://user-images.githubusercontent.com/403250/213483429-7fdd831f-9bdf-429f-90d8-33d67b1d62fa.mp4

Reply:
Yes, this was not working well 
I just added margin-top:0